src/main/java/io/github/moulberry/notenoughupdates/util/DiscordMarkdownBuilder.java

```diff
@@ -33,7 +33,19 @@ public DiscordMarkdownBuilder category(String name) {
 	}
 
 	public DiscordMarkdownBuilder append(String key, Object value) {
-		builder.append("[").append(key).append("]").append("[").append(value).append("]").append("\n");
+		if (!key.isEmpty()) {
+			builder.append("[").append(key).append("]");
+		}
+		builder.append("[").append(value).append("]").append("\n");
+		return this;
+	}
+
+	public DiscordMarkdownBuilder checkLast() {
```

Comment:
I'm not sure that we need this method. The builder itself should never contain ` ``` `, that would always be added in `#toString`

Reply:
Also this is a pretty terrible name for what it does.

Reply:
> I'm not sure that we need this method. The builder itself should never contain `` ``` ``, that would always be added in `#toString`

the builder ends up containing ``` when I did .tostring to check it. I've fixed all this now